elastic/kibana
# [Maps] fix array values out of order in tooltips

Fixes https://github.com/elastic/kibana/issues/133310

Tooltip fields where getting fetches as [docvalue_fields](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-fields.html#docvalue-fields). Doc value fields sort arrays and do not preserve the original order.

This PR switches from docvalue_fields to fields API for fetching tooltip field properties.

To test
* in Console create index
  ```
  PUT test_array_values/
  {
    "settings": {
      "number_of_shards": 1,
      "number_of_replicas": 1
    },
    "mappings": {
      "properties": {
        "location": { "type": "geo_point"},
        "title": { "type": "text"}, 
        "results": {"type": "integer"}
      }
    }
  }

  POST test_array_values/_doc
  {
    "location": [ 19, 52 ],
    "title": "5, 4, 2, 1",
    "results": [ 5, 4, 2, 1]
  }
  ```
* Create data view for index
* Create new map and add "Documents" layer. Select data view created above
* Add tooltip fields
* Click feature in map and verify tooltip is displayed with properties selected and that array is in order.
